libipl/libipl.H

```diff
@@ -18,11 +21,50 @@ enum ipl_type {
 #define IPL_INFO    1
 #define IPL_DEBUG   2
 
+//IPL Error types
+enum ipl_error_type {
+	IPL_NO_ERR = 0,
+	IPL_HWP_ERR,
+	IPL_SBE_CHIPOP_ERR,
+	IPL_SBE_BOOT_FAIL,
+	IPL_PRI_PROC_NON_FUNC,
+	IPL_FSI_TGT_NOT_FOUND,
+	IPL_CFAM_ERR,
+};
```

Comment:
I thought we agreed that all error codes should have `IPL_ERR_` prefix.

Reply:
Sorry missed .